homeassistant/components/homekit/__init__.py

```diff
@@ -73,8 +73,8 @@ def handle_homekit_service_start(service):
 
 def get_accessory(hass, state, aid, config):
     """Take state and return an accessory object if supported."""
-    _LOGGER.debug('<entity_id=%s aid=%d config=%s>',
-                  state.entity_id, aid, config)
+    # _LOGGER.debug('<entity_id=%s aid=%d config=%s>',
```

Comment:
With improved logging you mean disable logging? 😛 

We should either remove the commented statements or uncomment them.

Reply:
It just spamed the log for more than a few accessories.
I though they might still be useful later, but I guess I can code them in later as well. I will remove them.